pkg/karmadactl/cmdinit/utils/examples.go

```diff
@@ -3,21 +3,25 @@ package utils
 import (
 	"fmt"
 
-	"github.com/lithammer/dedent"
-
 	"k8s.io/klog/v2"
 )
 
 const (
+	// https://github.com/karmada-io/karmada/blob/master/artifacts/agent
 	karmadaAgent = `---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: karmada-cluster
+---
```

Comment:
The namespace will be created automatically along with the `karmada-impersonator` now.